sdk/media/azure-media-livevideoanalytics-edge/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+
+# Release History
+
+-------------------
+
+## 0.0.1 (Unreleased)
+
+- Training day!
```

Comment:
Guess it's not training day anymore :p. For a first release, you don't need much since there is no "changes" from something else, I used to write simply "Initial release".